src/lfsr/lfsr_galois.veryl

```diff
@@ -0,0 +1,168 @@
+module lfsr_galois #(
```

Comment:
Could you add documentation comments which are started by `///` like below?
https://github.com/veryl-lang/std/blob/d993ff8f5c150b0be20736c5f9ddd3d3b708ae98/src/delay/delay.veryl#L1-L18

Module description, parameter description and port description can be added.
Rendered document becomes https://std.veryl-lang.org.